migrate-wordpress-directus/index.md

```diff
@@ -0,0 +1,245 @@
+
+## Steps to Migrate Content From WordPress
+
+Migrating from WordPress to Directus involves some steps, primarily focusing on data migration.
+**1. Exporting WordPress Data**
+Start by exporting your WordPress data. Here it is assumed you already have a Wordpresss site. WordPress offers a built-in export tool that generates an XML file of your content. However, for a more comprehensive export, especially for custom post types and metadata, we will be using plugin that can export to formats more conducive to manipulation like JSON. 
+On your Wordpress Admin, in the **Plugins** tab search for “**WP Import Export Lite”**. Install and activate it. 
+After the installation select “**WP Imp Exp**” on the tab then select “**Post**” and select the format JSON in “**Advanced Options**”. Now you can download the JSON file.
+
+![JSON export from Wordpress](wp-exp.png)
+
+
+**2. Designing Your Directus Schema**
+Before importing data into Directus, design your schema in Directus by creating collections (equivalent to WordPress's custom post types), fields (similar to WordPress's custom fields), and relationships. This step is critical and requires a deep understanding of your content structure to ensure the data is imported correctly into your Directus setup.
+To begin log in to Directus and create a collection with the name “Posts” 
+In an actual app you might have other pages asides the Posts, so you can go ahead and create the Collection for those.
+Now we will create the fields that will hold the data items. If you are using my JSON data create the following fields:
+
+![FIelds for Collection](fields-creation.png)
+
+:::info Box title
+*Note that the “Content” field has a Field type of* ***WYSIWYG****, the rest are* ***Input*** *type.*
+:::
+
+**3. Importing Data into Directus**
+After the JSON file is extracted and the schema is developed on Directus, you can go to your Collections page on Directus admin and you will see a button there to import and export. Just choose export and select the JSON file you downloaded. 
+
+![IMport in Directus Collection](import-items.png)
+
+
+The problem with this method is, if you have images in the post, you won’t be able to access them. This is the advantage of Directus having a REST API. This the RESTAPI we will be able to manipulate the JSON to extact the images and save to Directus Files so we can access the files in the post.
+To use the API we need to get the API token at User Directory → Administrator → Admin User → Token. Copy the token and Keep it somewhere, we will use it later.
+ 
+We will also need  some permissions to post items to our Collection. You can grant those permissions at **Settings** → **Access Control** → **Public.** Allow all Permissions or whatever permisions you prefer.
+
+![Set permission](permissions.png)
+
+## Developing the Script to Export to Directus
```

Comment:
```suggestion
## Creating Import Scripts
```

Important term change - exporting from WP, importing to Directus.

Reply:
For the Pages, the process is the same. That's why I didn't include it. But I have made some updates to include it.